internal/cmd/cmd.go

```diff
@@ -281,16 +331,28 @@ func (c *Cmd) Wait() error {
 	if ioErr == nil {
 		ioErr, _ = c.stdinErr.Load().(error)
 	}
+	if err == nil {
+		err = ioErr
+	}
```

Comment:
Should we maybe wrap the errors together instead?

Reply:
as in:
```go
if err != nil {
  err := fmt.Errorf("multiple errors during IO copy: %w; %w", err, cerr)
}
```